ephypype/tests/test_spectral.py

```diff
@@ -0,0 +1,81 @@
+"""Test spectral."""
+
+import numpy as np 
+
+import os
+from ephypype.spectral import (compute_spectral_connectivity,
+                               compute_and_save_multi_spectral_connectivity)  # noqa
+
+import ephypype
+
+print(ephypype.__path__)
+
+print (ephypype.__version__)
+
+#data_path = mne.datasets.sample.data_path()
+#raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+
+time_length = 10000
+nb_ROI = 10
+
+nb_trials = 50
+
+sfreq = 500
+
+fmin = 0.1
+fmax = 300
+
+ts_mat = np.random.rand(nb_ROI, time_length) * \
+    np.random.choice([-1, 1], size=(nb_ROI, time_length))
+
+          
+ts_mat_trials = np.random.rand(nb_trials,nb_ROI, time_length) * \
+    np.random.choice([-1, 1], size=(nb_trials, nb_ROI, time_length))
+              
+##################################### testing compute_spectral_connectivity ###################################
+def test_compute_spectral_connectivity_one_trial():
+    
+    """Test compute_spectral_connectivity plv with one trial."""
+    try:
+        res_plv = compute_spectral_connectivity(data = ts_mat, con_method = "plv", mode = "multitaper", fmin = fmin, fmax = fmax, sfreq = sfreq, gathering_method='mean')
+    except SystemExit:
```

Comment:
There should be no `try-except` code in your tests ... you should use assert statements and pytest

Reply:
how do you assert a function that return sys.exit exception? I tried to find but couldn't

Reply:
you use `pytest.raises`